### ISSUE TITLE
SSR support?

### ISSUE THREAD

**Issue by chin2km** [dahliacreative/react-device-breakpoints#6]
This library works well on CSR but breaks in SSR mode.
Do you have plans to support SSR?

**Comment by dahliacreative** [comment]
Yes I plan to support SSR in the future when I get some spare time. If you have any specific errors when using it with SSR please post them here. 

That being said we don’t have access to the window on the server so it won’t actively render the correct element for the users window size but just render out the first breakpoint definition.

We could however except a paean which could be sent from the client to the server as a header request maybe.

I’ll update on any progress I make.
